Arreglos generales usabilidad

adriantd3 opened the pull request:
- El lector de pantalla es más corto y conciso al decir el nombre de una playlist desde una card.
- Añadido headers de sección (por ejemplo Mis listas: cuando estas en la pagina de tus listas).
- El lector de pantalla ahora avisa correctamente cuando se abre un dialogo (excepto en las vistas de playlist. Ahí el audio se corta y no sé por que)
- Cambios de la privacidad de check a radio buttons para que sea más usable y accesible.
- Arreglado el title del boton de mute.

adriantd3 committed:
Cards accesibilidad arreglado

adriantd3 committed:
Activar/Silenciar volumen title arreglado

adriantd3 committed:
Modals PlaylistInfo

adriantd3 committed:
Arreglos modals

adriantd3 committed:
Privacidad con radiobuttons

adriantd3 committed:
Arreglos trackList y accesible

adriantd3 committed:
Accesibilidad playlistInfo y addSongButton

adriantd3 committed:
Arreglos generales

- CardsGrid ahora tiene como rol "grid".
- Establecidos arias attributes para cuando hay un error en listmodal.
- Cambiados aria-label y description para listas y busqueda.
- Arreglos navButton para que informe la página actual.
- Arreglado boton de reproducir cancion songbutton en modo claro.
- Arreglado title de boton de reproducir en musicplayer.

adriantd3 committed:
Arreglado title y aria boton de seguir

adriantd3 committed:
Music Pllayer información de la cancion actual

adriantd3 committed:
Texto alternativo imagenes playlist

adriantd3 commented:
- En el primer commit he arreglado un error en el cual un header (h4) estaba contenido dentro de un párrafo(p). Tambien se ha hecho que el lector de pantalla lo lea correctamente.
- En el segundo commit se ha hecho que el lector mencione la información de la playlist y que cuando pase sobre el boton de configuración lo lea correctamente. Tambien se ha arreglado un problema por el cual cuando se llegaba al boton de añadir canción, se leía el contenido 2 veces.
- En el commit de arreglos generales se han cambiado muchas cosas, estan en la descripcion.
- En el cuarto commit se ha añadido title y aria-pressed al boton de seguir para que el lector lo reconozca correctamente. Aria-pressed para que el lector diga "boton presionado" cuando esta activo.
- En el quinto commit se ha arreglado el div de musicPlayer para que mencione el nombre de la cancion actual y el artista con tabs
- En el ultimo se ha corregido el fallo de accesibilidad de imágen sin alt en playlistinfo

DavidBuenoCarmona commented:
He visto que el nombre de la canción y el artista los dice en la misma sección, pero luego pasa al nombre de la canción otra vez, eso debe ser así?

adriantd3 committed:
Varios cambios

- Añadido label de cargando y aria-busy a secciones.
- Arreglado error en búsquedas, no leía recomendaciones para ti.
- Cambiada descripcion de songButton.
- Cambiada duración para que diga correctamente los minutos y segundos.
- Titles de botones de añadir y configuracion

adriantd3 committed:
Arreglo contraste e imagen placeholder en lista

adriantd3 committed:
Fixes de listModal y cardsGrid

adriantd3 committed:
Orden de headers actualizados

adriantd3 committed:
Cambios colores y arreglos boton mostrar mas

adriantd3 committed:
small fix

adriantd3 committed:
last fixes

adriantd3 committed:
merge
